assets/charts/components/prometheus-operator/templates/grafana/dashboards-1.14/kubelet.yaml

```diff
@@ -191,7 +191,7 @@ data:
                         "tableColumn": "",
                         "targets": [
                             {
-                                "expr": "sum(kubelet_running_pod_count{cluster=\"$cluster\", job=\"kubelet\", metrics_path=\"/metrics\", instance=~\"$instance\"})",
+                                "expr": "sum(kubelet_running_pods{cluster=\"$cluster\", job=\"kubelet\", metrics_path=\"/metrics\", instance=~\"$instance\"})",
```

Comment:
Is this change reported upstream?

Reply:
I wonder where should I report this issue because upstream seems to have removed the chart: https://github.com/grafana/helm-charts/tree/main/charts/grafana.

Reply:
I think source of the dashboards is https://github.com/kubernetes-monitoring/kubernetes-mixin, no?

Reply:
It is fixed in the upstream: https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/499